src/js/vic-v2/config/form.js

```diff
@@ -190,6 +191,7 @@ const formConfig = {
               }
             }), {
               'ui:field': PhotoField,
+              'ui:reviewDescription': photoReviewDescription,
```

Comment:
A simpler path than building this into the common schemaform code might be to detect if we're in review mode in the PhotoField component (it's in `this.props.formContext.reviewMode`), and return this description there. We've done that in some other custom fields.